Libraries/Text/Text/RCTTextView.m

```diff
@@ -273,6 +275,32 @@ - (void)didMoveToWindow
 
 #pragma mark - Accessibility
 
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+
+// This code is here to cover for a mismatch in the what Labels and Values mean in RN vs in Text components on MacOS
+// In MacOS a text element will always read its Value, but will only read it's Label if it's value is set
+// In RN (iOS & Android) a text element will only read it's Value if it has no Label, and will always read its Label
+// This code replicates the expected RN behavior in MacOS by: 
+// 1) Setting the Value = the RN Label if one exists and setting it equal to the text's contents otherwise.
+// 2) Making sure that its Label is always nil, so that it doesn't read out the label twice.
+
+- (void)setAccessibilityLabel:(NSString *)label
+{
+  _accessibilityLabel = label;
+}
+
+- (NSString *)accessibilityValue
+  {
+  if(_accessibilityLabel){
+    return _accessibilityLabel;
+  }
+  NSString *superAccessibilityLabel = [super accessibilityLabel];
```

Comment:
Still a bit odd to call super on a method outside of the override of that specific method?

Reply:
I agree that it's odd. It's less odd than normal though because in this instance accessibilityValue is doing the job of accessibilityValue. Though we only kept the super around because of worries removing it might break some corner cases around nesting textViews. I'm taking a look at those cases to see if we can remove the super.

Reply:
After poking around it looks like if you have two nested text elements on both iOS and MacOS – with or without calling super– the inner text elements accessibilityLabel has no effect. If the outer text element has a label that label and nothing else will be read. If the outer text does not have a label VO will read the outer text and then the inner text disregarding the inner label.

I think that's enough to say we don't need the super. In all other cases it should either be getting it's label from the JS which is covered by setAccessibilityLabel/_accessibilityLabel, or we can just return the text.